images/linux/scripts/installers/packer.sh

```diff
@@ -5,7 +5,7 @@
 ################################################################################
 
 # Install Packer
-PACKER_VERSION=$(curl -s https://checkpoint-api.hashicorp.com/v1/check/packer | jq -r .current_version)
+PACKER_VERSION=$(curl -s https://api.releases.hashicorp.com/v1/releases/packer/latest | jq -r .version)
 curl -LO "https://releases.hashicorp.com/packer/${PACKER_VERSION}/packer_${PACKER_VERSION}_linux_amd64.zip"
```

Comment:
Replace curl to the helper function `download_with_retries` [source $HELPER_SCRIPTS/install.sh]

Reply:
updated